rust/repositories.bzl

```diff
@@ -493,6 +511,32 @@ def _load_rust_compiler(ctx):
 
     return compiler_build_file
 
+def _load_rust_src(ctx):
+    """Loads the rust source code. Used by the rust-analyzer rust-project.json generator.
```

Comment:
Can you add type annotations to the docstring? 🙏 😅

Reply:
None of the _functions in this file have type annotations, leaving for another PR

Reply:
That's just because I hadn't gotten to this file when updating all the docs 😅. I don't think anyone should be expected to update an entire file if they touch it, but I do think new macros should have them.